app/scripts/app.js

```diff
@@ -187,6 +163,20 @@ angular.module('oncokbApp').run(
                 return defer.promise;
             }
 
+            function getEvidenceLevels() {
+                DatabaseConnector.getEvidenceLevels(function(data) {
+                    _.forEach(data.levels, function(level) {
+                        var levelCode = level.levelOfEvidence.replace('LEVEL_', '');
+                        $rootScope.meta.levelsDesc[levelCode] = level.description;
+                        $rootScope.meta.levelsDescHtml[levelCode] = level.htmlDescription;
+                        $rootScope.meta.colorsByLevel[levelCode] = level.colorHex;
+                    });
+                }, function() {
+                    dialogs.error('Error', 'Failed to load evidence levels information. Please Contact developer and stop curation.');
+                });
+            }
+
+            getEvidenceLevels();
```

Comment:
This is called in a free form. I think it needs to be guaranteed executed before rendering other pages

Reply:
I tested on other controllers by setting breaking points. Once the app.js is done, other pages will start to render.